tests/Fixer/ControlStructure/SwitchContinueToBreakFixerTest.php

```diff
@@ -424,8 +427,10 @@ public static function provideFixCases(): iterable
                     continue;
             }
             ',
-            [
-                '<?php
+        ];
+
+        yield [
+            '<?php
```

Comment:
looks like there was some mistake in 2 cases merged into 1 weird [string, string, [string, string]],
but now the case-description [here](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8544/files#diff-0690dd90f0f6c79b51a0cf61fdd8af4a1647abdd4dc304f82b6846052d1aaf88R415) doesn't make sense

Reply:
#8546